packages/api/cms-api/src/auth/guards/comet.guard.ts

```diff
@@ -8,7 +8,8 @@ import { isObservable, lastValueFrom } from "rxjs";
 import { CurrentUserInterface } from "../current-user/current-user";
 import { allowForRoleMetadataKey } from "../decorators/allow-for-role.decorator";
 
-function CometAuthGuard(type?: string | string[]): Type<IAuthGuard> {
+export function createCometAuthGuard(type?: string | string[]): Type<IAuthGuard> {
+    @Injectable()
     class GlobalAuthGuard extends AuthGuard(type) implements CanActivate {
```

Comment:
Shouldn't this be name CometAuthGuard as well?

Reply:
This is changed now.